components/Componenten/input-stepper/input-stepper.njk

```diff
@@ -0,0 +1,16 @@
+{% set localId = generateLocalId(prefix, id) %}
+
+<div class="dso-input-stepper">
+  {% if label %}
+    <label for="{{ localId }}" class="control-label">
+      {{ label }}
+    </label>
+  {% endif %} 
+  {% render '@button', {type: 'button', modifier: 'link', label: minusButtonLabel, icon: 'minus-square', iconOnly: 'true', disabled: minusButtonInactive} %}
+  <input type="number" 
+          id="{{ id }}" 
+          readonly
```

Comment:
Waarom eigenlijk readonly? Je kunt 'n namelijk wel focusen, dus dan zou ik er ook eigenlijk best in willen typen! Zeker als het dan toch met `min` en `max` wordt gecombineerd zodat ik als gebruiker geen fouten kan maken.

Reply:
Maar als je zelf kunt invoeren, zal de styling van de input daar op aangepast moeten worden (lees: wel bordertje)? Dat moeten we dan met Bram kortstluiten.